### PR TITLE
Bill Model

### PR DESCRIPTION
**Pull request by preet4737**

### CODE REVIEW

File: canteenWeb/models.py [hunk 1]
```diff
@@ -40,6 +40,10 @@ class TeacherProfile(models.Model):
 #####################
 
 
+class Category(models.Model):
```
[the remark this answers is not in the document]
Comment (vixrant) [reply canteenWeb/models.py]: https://xd.adobe.com/spec/65350c4d-670e-4907-6870-786df666f904-1304/screen/5e16c8c6-6d4b-44cb-a056-00d5dfc567bd/Add-Category/

It was in the frontend design...

File: canteenWeb/urls.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
 router = routers.DefaultRouter()
```
[the remark this answers is not in the document]
Comment (vixrant) [reply canteenWeb/urls.py]: What's the difference though?

File: canteenWeb/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sahiljajodia01) [reply canteenWeb/models.py]: Ok

File: canteenWeb/urls.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sahiljajodia01) [reply canteenWeb/urls.py]: How are you gonna specify URLs to frontend people? Just wanted to be clear for this.

**Pull request merged**